main/src/ca/uwaterloo/flix/language/phase/jvm/JvmOps.scala

```diff
@@ -816,6 +816,40 @@ object JvmOps {
     typesOf(root).flatMap(tpe => getTagsOf(tpe)(root, flix))
   }
 
+  /**
+    * Returns the set of ref types in `types` without searching recursively.
+    */
+  def getRefsOf(types: Iterable[MonoType])(implicit flix: Flix, root: Root): Set[BackendObjType.Ref] =
```

Comment:
Do you want to take a stab at collecting these in the eraser? Or only later?

Reply:
I think its better to do once BackendType is fully formed

Reply:
Any performance impact?